### PR TITLE
App strings reworked 3

### PR DESCRIPTION
**Pull request by comradekingu**
Seems the use of ending dot for error messages isn't consistent, but that is another matter.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -910,10 +910,10 @@
     <string name="input_key">Input key (in hexadecimal)</string>
     <string name="generate_key">Generate</string>
     <string name="failed_to_save_key">Could not save key.</string>
-    <string name="failed_to_initialize_key_store">Could not initialize App Manager keystore.</string>
-    <string name="invalid_aes_key_size">Invalid key size for AES encryption. Key size can be either 128 or 256 bits.</string>
+    <string name="failed_to_initialize_key_store">Could not initialize App Manager KeyStore.</string>
+    <string name="invalid_aes_key_size">Use 128 or 256 bit key size for AES encryption isntead.</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:914]:
```suggestion
    <string name="invalid_aes_key_size">Use 128 or 256 bit key size for AES encryption instead.</string>
```

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1050,14 +1050,14 @@
...
-    <string name="confirm_import_keystore">Are you sure want to replace the existing KeyStore? This action cannot be undone.</string>
+    <string name="confirm_import_keystore">Replace the existing KeyStore? This action cannot be undone.</string>
     <string name="failed">Failed</string>
     <string name="pref_rules_msg">Instant blocking, import/export/remove rules, etc.</string>
     <string name="saf">SAF</string>
-    <string name="pref_layout_orientation">Layout orientation</string>
-    <string name="orientation_follow_locale">Follow locale</string>
-    <string name="orientation_left_to_right">Left to right</string>
-    <string name="orientation_right_to_left">Right to left</string>
+    <string name="pref_layout_orientation">Text direction</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:1059]: Not 100% sure, and this is probably not technically correct. It is however more clear.
Layout orientation sounds like it changes to top-to-bottom, or the aspect ratio to me.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1001,8 +1001,8 @@
     <string name="pref_log_line_period_error">Please enter an integer between 1 and 1000.</string>
     <string name="pref_log_write_period_summary">When recording, write to SD card every %1$d lines.</string>
     <string name="pref_log_write_period_title">Write Period</string>
-    <string name="pref_show_timestamp_summary">Show process id and timestamp when expanded.</string>
-    <string name="pref_show_timestamp_title">Show Pid &amp; Timestamp</string>
+    <string name="pref_show_timestamp_summary">Show process ID and timestamp when expanded.</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:1004]: What is this the timestamp of? Explanation no clearer than the title in that regard.

**Comment by MuntashirAkon** [comment]
I suggest you make up your mind before making a PR as I have to waste my important hours analysing and replying to them. If you don't know a technical term, avoid fixing them or look them up. Fixing strings is a very low priority issue right now as we have much bigger issues.

Only make a PR when you're sure that everything's in place. Thanks.

**Comment by comradekingu** [comment]
@MuntashirAkon Anything wrong with this?

**Pull request closed without merging**